typescript/processors/example-write-set-change-processor/processor.ts

```diff
@@ -1,62 +1,22 @@
-import * as services from "@/aptos/indexer/v1/raw_data_grpc_pb";
-import {
-  GetTransactionsRequest,
-  TransactionsResponse,
-} from "@/aptos/indexer/v1/raw_data_pb";
-import { Config } from "@/utils/config";
+import { aptos } from "@aptos-labs/aptos-indexer-protos";
+import { Config } from "../../utils/config";
 import { Timer } from "timer-node";
 import { exit } from "process";
-import { ChannelCredentials, CallCredentials, Metadata } from "@grpc/grpc-js";
+import { ChannelCredentials, Metadata } from "@grpc/grpc-js";
 import { parse as pgConnParse } from "pg-connection-string";
 import { program } from "commander";
-import { createDataSource } from "@/processors/example-write-set-change-processor/models/data-source";
-import { Event } from "@/processors/example-write-set-change-processor/models/Event";
-import {
-  INDEXER_NAME,
-  parse,
-} from "@/processors/example-write-set-change-processor/event-parser";
-import { NextVersionToProcess } from "@/utils/common_models/NextVersionToProcess";
-
-// A hack to override the http2 settings
```

Comment:
Why delete this?

Reply:
no, this was not related to TLS. This is the settings we needed for the http2 flow control. Without these settings, the grpc link is very slow. With this, the link has similar performance as Python

Reply:
Okay got it, I'll try to make this work with TLS then.

Reply:
I'm having a bit of trouble getting this working, asking about it here: https://github.com/grpc/grpc-node/issues/2565.